### PR TITLE
Move web.config and use ANCM

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Review: @Tratcher 

RC2 Approval: @Eilon @DamianEdwards

### CODE REVIEW

File: samples/SignalRSample.Web/web.config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <system.webServer>
+    <handlers>
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
+    </handlers>
+    <aspNetCore processPath="./SignalRSample.Web.exe" arguments="" forwardWindowsAuthToken="false" />
```
Comment (moozzyk) [inline samples/SignalRSample.Web/web.config]: I would just remove arguments.

File: samples/SignalRSample.Web/project.json [hunk 2]
```diff
@@ -12,15 +12,20 @@
...
   "frameworks": {
     "net451": {},
     "netstandardapp1.5": {
       "imports": [
         "dnxcore50"
-      ]
+      ],
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-*",
+        "System.Runtime.Serialization.Primitives": "4.1.1-*"
```
Comment (moozzyk) [inline samples/SignalRSample.Web/project.json]: Isn't it part of NETStandard.Library?

Comment (JunTaoLuo) [inline samples/SignalRSample.Web/project.json]: Don't think so, I had to add it specifically.

File: samples/SignalRSample.Web/web.config (hunk 1 above)
Comment (JunTaoLuo) [inline samples/SignalRSample.Web/web.config]: makes sense, I'll do this clean up when I merge. Also change the direction of the slashes.

**Comment by moozzyk** [comment]
:ship: :it:

**Pull request merged**